go.mod

```diff
@@ -5,29 +5,25 @@ go 1.17
 require (
```

Comment:
can't comment on a commit message but this change is really nice. i think you real genius here was figuring out we didn't need need ethereum curve and that was the real baggage we'd been caring around unnecessarily?

Reply:
yep, that every word is true.

TBH I've spent a half day to re-implement that curve (repeating myself)... just to find out that it fails on Golang's TLS implementation... so, the further search led me to understand that we use the P256 curve which has a misleadingly close name to S256.

Unfortunately for us, the `go-ethereum` crypto library is as cryptic as it could be when naming curves - one should look at the curve numbers and compare them to industry standards in order to guess, rather than just read their name.